docs/simplified_im_b/master04-sim_data_types.adoc

```diff
@@ -149,8 +167,21 @@ The following shows the SDT serial formats for an `S_DV_QUANTITY` instance:
 {
     "a/b/c/d|magnitude": 125,
     "a/b/c/d|units":    "mm[Hg]",
-    "a/b/c/d|xx":       "xx_val", -- other DvQ fields
-    "a/b/c/d|yy":       "yy_val"
+    "a/b/c/d|precision":    1,
+    "a/b/c/d|normal_status":  "H",
+    "a/b/c/d|magnitude_status": "~",
+    "a/b/c/d/_normal_range/lower|magnitude": 2.5,
```

Comment:
Why does `_normal_range` start with an '_' here? Isn't `normal_range` just a normal attribute of a `DV_ORDERED`?

Reply:
My understanding is that the underscore is used on a few RM attributes to avoid potential naming collisions with archetyped nodes e.g it might be the case that a potential archetype node is called 'normal status' and gets compressed to 'normal_status'.